openmetadata-ui/src/main/resources/ui/src/utils/GlobalSettingsUtils.tsx

```diff
@@ -288,7 +292,11 @@ export const getGlobalSettingMenuItem = (
 } => {
   const subItems = children
     ? children
-        .filter((menu) => menu.isProtected)
+        .filter((menu) =>
+          menu.isAdmin
+            ? menu.isProtected
+            : menu.isProtected && isUndefined(menu.isAdmin)
+        )
```

Comment:
We don't need this logic if we just use the `isProtected` flag.

Reply:
Done